isotovideo

```diff
@@ -120,6 +120,11 @@ $bmwqemu::vars{BACKEND} ||= "qemu";
 # This allows further structuring the test distribution collections with
 # multiple distributions or flavors in one repository.
 $bmwqemu::vars{PRODUCTDIR} ||= $bmwqemu::vars{CASEDIR};
+
+# as we are about to load the test modules store the git hash that has been
+# used. If it is not a git repo fail silently, i.e. store an empty variable
+chomp($bmwqemu::test_git_hash = qx{git rev-parse HEAD});
```

Comment:
there is no global results.json. results are per test - while the status is for the whole test run. And these files are read by the worker and transferred back. vars.json is just kept for documentation - and removed after 14 days.